source/config/__init__.py

```diff
@@ -19,6 +25,18 @@
 import easeOfAccess
 import winKernel
 
+#Choices for numbers combo box in Voice settings.
+
+digitChoices=[
+	# Translators: Choice in a combo box for speaking numbers by word.
+	"Words",
+	# Translators: Choice in a combo box for speaking numbers as digits.
+	"Digits",
+	# Translators: Choice in a combo box for speaking numbers as double digits.
+	"Double Digits",
+	# Translators: Choice in a combo box for speaking numbers as triple digits.
+	"Triple Digits"]
+
 def validateConfig(configObj,validator,validationResult=None,keyList=None):
 	"""
 	@deprecated: Add-ons which need this should provide their own implementation.

```

Comment:
I assume you mean remove it; it's already deprecated. Sure, but please 
submit a separate PR for that.

Btw, the strings you intend to be translatable aren't translatable. You 
need to enclose translatable strings in _(); e.g. _("Words").